pt-BR/mail/chrome/messenger/pbMessenger.dtd

```diff
@@ -138,223 +138,223 @@
-<!ENTITY msgAnnotation.title                         "Edit">
-<!ENTITY saveMessageEditing.button.label             "Save">
+<!ENTITY msgAnnotation.title                         "Editar">
+<!ENTITY saveMessageEditing.button.label             "Salvar">
 <!ENTITY saveMessageEditing.button.accesskey         "S">
-<!ENTITY saveMessageEditingAsCopy.button.label       "Save As Copy">
+<!ENTITY saveMessageEditingAsCopy.button.label       "Salvar como cópia">
 <!ENTITY saveMessageEditingAsCopy.button.accesskey   "a">
-<!ENTITY cancelMessageEditing.button.label           "Cancel">
+<!ENTITY cancelMessageEditing.button.label           "Cancelar">
 <!ENTITY cancelMessageEditing.button.accesskey       "C">
 
 <!-- inline message editing toolbar -->
-<!ENTITY boldEditMsgToolbarCmd.tooltip               "Bold">
-<!ENTITY italicEditMsgToolbarCmd.tooltip             "Italic">
-<!ENTITY underlineEditMsgToolbarCmd.tooltip          "Underline">
-<!ENTITY strikethroughEditMsgToolbarCmd.tooltip      "Strikethrough">
-<!ENTITY cropTextEditMsgToolbarCmd.tooltip           "Crop Text">
-<!ENTITY insertLinkEditMsgToolbarCmd.tooltip         "Insert new link or edit selected link's properties">
-<!ENTITY insertListEditMsgToolbarCmd.tooltip         "Apply or remove bulleted or numbered list">
-<!ENTITY noneMsgToolbarCmd.label                     "None">
+<!ENTITY boldEditMsgToolbarCmd.tooltip               "Negrito">
+<!ENTITY italicEditMsgToolbarCmd.tooltip             "Itálico">
+<!ENTITY underlineEditMsgToolbarCmd.tooltip          "Sublinhado">
+<!ENTITY strikethroughEditMsgToolbarCmd.tooltip      "Tachado">
+<!ENTITY cropTextEditMsgToolbarCmd.tooltip           "Cortar texto">
+<!ENTITY insertLinkEditMsgToolbarCmd.tooltip         "Inserir novo link ou editar as propriedades do link selecionado">
+<!ENTITY insertListEditMsgToolbarCmd.tooltip         "Aplicar ou remover lista numerada ou com marcadores">
+<!ENTITY noneMsgToolbarCmd.label                     "Nenhuma">
 <!ENTITY noneMsgToolbarCmd.accesskey                 "N">
-<!ENTITY listBulletMsgToolbarCmd.label               "Bulleted">
+<!ENTITY listBulletMsgToolbarCmd.label               "Com marcadores">
 <!ENTITY listbulletMsgToolbarCmd.accesskey           "b">
-<!ENTITY listNumberedMsgToolbarCmd.label             "Numbered">
+<!ENTITY listNumberedMsgToolbarCmd.label             "Numerada">
 <!ENTITY listnumberedMsgToolbarCmd.accesskey         "m">
-<!ENTITY insertHTMLEditMsgToolbarCmd.tooltip         "Insert HTML or edit selected HTML">
-<!ENTITY switchBetweenNormalAndCode.tooltip          "Switch between Normal and Code views">
-<!ENTITY outdentEditMsgToolbarCmd.tooltip            "Outdent text (move left)">
-<!ENTITY indentEditMsgToolbarCmd.tooltip             "Indent text (move right)">
+<!ENTITY insertHTMLEditMsgToolbarCmd.tooltip         "Inserir HTML ou editar HTML selecionado">
+<!ENTITY switchBetweenNormalAndCode.tooltip          "Alternar entre exibições Normal e de Código">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY switchBetweenNormalAndCode.tooltip          "Alternar entre exibição Normal e de Código">